src/convertToObject.js

```diff
@@ -5,8 +5,28 @@
  *
  * @return {object}
  */
+
 function convertToObject(sourceString) {
-  // write your code here
+  const StyleObj = {};
```

Comment:
```suggestion
  const styleObj = {};
```
camelCase for variables
We use PascalCase for classes (or constructor functions) in JS